Sources/PubNubChat/Storage/CoreData/PubNubManagedMember.swift

```diff
@@ -136,6 +137,7 @@ extension PubNubManagedMember: ManagedMemberEntity {
     self.id = member.id
     
     if let isPresent = member.presence?.isPresent {
+      print("*** Update isPresent for member to \(isPresent)")
```

Comment:
Is it just for debug purpose or should stay in release?

Reply:
Nope I'm just dumb and wanted to push this out before I ate dinner :(